tests/views/test_access_restriction.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Access restriction tests."""
+
+import django.urls
+
+import urls
+import views
+
+import django_tests_base
+
+
+class AccessRestrictionTests(django_tests_base.DjangoTestsBase):
```

Comment:
Also test that the ones mapped to False are available to non-admins by checking that the status_code is not 403? (could be 400 because you won't set up parameters correctly)

Reply:
Done.